de/ros/ros2_comm.md

```diff
@@ -119,7 +119,7 @@ To install ROS 2 and its dependencies:
 1. Some Python dependencies must also be installed (using **`pip`** or **`apt`**):
 
    ```sh
-   pip install --user -U empy pyros-genmsg setuptools
+   pip install --user -U empy==3.3.4 pyros-genmsg setuptools
```

Comment:
@frede791 REVERTING. Note, you never update anything outside the **/en/** tree - that's the English version. The translation system automatically syncs the changes to these paths like /de/ for other languages. If I left this it would make merging the internationalised version hard:

```suggestion
   pip install --user -U empy pyros-genmsg setuptools
```